sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestBySqlAggregator.java

```diff
@@ -122,12 +122,20 @@ public Aggregation toDruidAggregation(
         );
         break;
       case 3:
+        int maxStringBytes;
+        try {
+          maxStringBytes = RexLiteral.intValue(rexNodes.get(2));
```

Comment:
From the docs, it seems that `maxBytesPerString` is always the second argument, i.e. `rexNodes.get(1)`

Reply:
For `EARLIEST_BY/LATEST_BY`, it's the third argument. For non `_BY` aggregators, it's second argument.

Reply:
I fixed the error message though.

Reply:
Thanks for the clarification, I was looking at an older doc in the original PR.